src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -1164,6 +1317,10 @@ private static Args ParseFromFile(string fileName)
             {
                 verifyLiveSize = true;
             }
+            else if (word == "compute")
```

Comment:
NIT: maybe doing a case insensitive ordinal comparison here might be both performant and be less error prone in case the users provide the word in a different format: 

```csharp
else if (string.Equals(word, "compute", StringComparison.OrdinalIgnoreCase))
```

Reply:
Well, we aren't doing this in other places - even the command line option parsing is case sensitive, despite a comment stating the opposite.